recipes/configs/phi3/mini_full.yaml

```diff
@@ -65,6 +65,7 @@ memory_efficient_fsdp_wrap: False
 dtype: bf16
 
 # Logging
+output_dir: /tmp/phi3_full_finetune_output
```

Comment:
yeah we should keep a consistent naming scheme for output directories across configs